src/features/session_replay/instrument/index.js

```diff
@@ -44,10 +54,14 @@ export class Instrument extends InstrumentBase {
   }
 
   async #startRecording (mode) {
+    if (this.featAggregate) {
```

Comment:
Also need a check in here to see if recorder exists, and if recorder is recording --> early return